packages/components/src/Card/Card.module.css

```diff
@@ -88,3 +94,106 @@
   border-color: var(--color-purple-400);
   background-color: var(--color-purple-100);
 }
+
+.aiMoment {
+  --clr-1: oklch(80.15% 0.101 199.35deg) 0%;
+  --clr-2: oklch(76.41% 0.098 175.22deg) 20%;
+  --clr-3: oklch(80.15% 0.101 199.35deg) 41%;
+  --clr-4: oklch(72.34% 0.143 315.73deg) 100%;
+  --animated-border-width: 2px;
+
+  background-color: transparent;
+  position: relative;
+  border: 1px solid transparent;
+  border-radius: calc(var(--border-borderless-border-radius) + 1px);
+}
+
+/* This absolute element is used to give the illusion of a gradient border without expanding the size of the Well */
+.aiMomentBackground {
+  margin: var(--animated-border-width);
+  box-sizing: border-box;
+  position: absolute;
+  z-index: -1;
+  inset: 0;
+  display: block;
+  border-radius: var(--border-borderless-border-radius);
```

Comment:
```suggestion
  border-radius: calc(var(--border-borderless-border-radius) - 2px);
```

I don't want this to come off as too picky, but we can make the border radius look more natural if we do this!

Before:
![Screenshot 2025-04-14 at 2 05 37 pm](https://github.com/user-attachments/assets/9a9f6bdb-d4bf-4e76-a873-f004b1d31556)

After:
![Screenshot 2025-04-14 at 2 05 44 pm](https://github.com/user-attachments/assets/8e783185-6589-4f08-a357-f0d9d55dab6f)

Reply:
I just played around with the `calc` to see what looked best, but here's some quick math for the reason why nested border radius need a bit of extra work to look good: https://www.30secondsofcode.org/css/s/nested-border-radius/

Reply:
A better solution I just saw is to use `--animated-border-width`

Reply:
Nono, I agree, this looks much better and really more of my mistake :) Long live pedantry in all things border widths and radii 🥳 `--animated-border-width` should do the trick nicely

Reply:
Cool those border should be nice and properly rounded now :) appreciate the eagle eyes 🐦‍⬛